src/pretix/presale/forms/checkout.py

```diff
@@ -152,6 +152,9 @@ def _label(self, event, item_or_variation, avail, override_price=None):
             n += ' – {}'.format(_('SOLD OUT'))
         elif avail[0] < 100:
             n += ' – {}'.format(_('Currently unavailable'))
+        else:
+            if avail[1] is not None:
+                n += ' - {}: {}'.format(_('Current availability'), avail[1])
```

Comment:
I would recommend to use the same dash as in the other cases (`–` not `-` ("–" not "-"..) monospace killing the differece so probably overlooked), looks weird if it's displayed one the same page

Reply:
Also I personally feel more comfortable if we embed the message in a `small`-tag like it's done for "full-grown" articles - that would again mean we should also modify the other two cases..

Reply:
Yeah, it should be ``–``. I'm less sure about the ``<small>`` tag but I admit it looks nice – we'd need to be consistent here, though, as you noted.

Reply:
Also, maybe we choose the wording to be
```
n += ' – {}'.format(_('%(num)s currently available') % {'num': avail[1]})
```
It looks a bit worse in code, but it (a) leaves the order of text and number up to the translator and (b) re-uses the exact same translation string as the other place where we do this, leading to a more consistant language and less work for translators.